qcschema/dev/properties/cc_properties.py

```diff
@@ -0,0 +1,101 @@
+"""
+The complete list of Coupled-Cluster properties.
+"""
+
+cc_properties = {}
+
+cc_properties["ccsd_same_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The portion of CCSD doubles correlation energy from same-spin (i.e. triplet) correlations, without any user scaling."
+}
+
+cc_properties["ccsd_opposite_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The portion of CCSD doubles correlation energy from opposite-spin (i.e. singlet) correlations, without any user scaling."
+}
+
+cc_properties["ccsd_singles_energy"] = {
+    "type": "number",
+    "description": "The singles portion of the CCSD correlation energy. Zero except in ROHF."
+}
+
+cc_properties["ccsd_doubles_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The doubles portion of the CCSD correlation energy including same-spin and opposite-spin correlations."
+}
+
+cc_properties['ccsd_correlation_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The CCSD correlation energy."
+}
+
+cc_properties['ccsd_total_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The total CCSD energy (CCSD correlation energy + HF energy)."
+}
+
+cc_properties['ccsd_t_correlation_energy'] = {
```

Comment:
I vote for `ccsd_t__correlation_energy` or `ccsd_prt_pr_correlation_energy` so these are two-way translatable.

Reply:
I think from @berquist's comment from before going with the second option would be better. Is there a reason the second option is `ccsd_prt_pr_correlation_energy` vs. `ccsd_t_pr_correlation_energy`? It seems like the double use of the `pr` is redundant? Or are you using the two sets of `pr` as an open and close parentheses?

Reply:
I updated the keywords to follow the `ccsd_prt_pr_*` naming convention.